vertexai/genai/client_test.go

```diff
@@ -259,10 +259,31 @@ func TestLive(t *testing.T) {
 		if err != nil {
 			t.Fatal(err)
 		}
-		checkMatch(t, responseString(res), "(it's}|weather) .*cold")
+		checkMatch(t, responseString(res), "(it is|it's}|weather) .*cold")
 	})
 }
 
+func TestLiveREST(t *testing.T) {
```

Comment:
This is fine for this CL.
Ideally we should execute all the tests in TestLive for both transports.

Reply:
Fair enough - I was mostly worried about the test time - it's already taking a bit. We can do it in a followup

Reply:
We can do them concurrently.
I'll own it.